pvlib/clearsky.py

```diff
@@ -195,7 +195,9 @@ def ineichen(time, location, linke_turbidity=None,
     
     cos_zenith = tools.cosd(ApparentZenith)
     
-    clearsky_GHI = cg1 * I0 * cos_zenith * np.exp(-cg2*AMabsolute*(fh1 + fh2*(TL - 1))) * np.exp(0.01*AMabsolute**1.8)
+    clearsky_GHI = ( cg1 * I0 * cos_zenith *
```

Comment:
I only intended to change the API related names for now. I would like to see the internal variables get updated, but I think we can do this in smaller chunks.